src/AppInstallerCLICore/Commands/RootCommand.cpp

```diff
@@ -110,6 +110,24 @@ namespace AppInstaller::CLI
                 }
             }
         }
+
+        void OutputAdminSettings(Execution::Context& context)
+        {
+            auto info = context.Reporter.Info();
+            info << std::endl;

```

Comment:
nit: if OutputGroupPolicies above is not putting an empty line in the end, this should be moved to end of OutputGroupPolicies function. Otherwise, this line is probably not needed. please also remove the 
`auto info = context.Reporter.Info();` line above too.